source/_integrations/amcrest.markdown

```diff
@@ -424,4 +424,29 @@ amcrest:
       - ptz_preset
 ```
 
+## Example Automation to Detect Button Presses on AD110 and AD410 Doorbells
+
+Using this trigger in an automation will allow you to detect the press of the doorbell call button and create automations based upon it. 
+
+```yaml
+# Example automations.yaml entry
+alias: Doorbell Pressed
+description: "Trigger when Amcrest Button Press Event Fires"
+trigger:
+  - platform: event
+    event_type: amcrest
+    id: doorbell
+    event_data:
+      event: CallNoAnswered
+      payload:
+        action: Start
+condition: []
+action:
+  - type: flash
+    entity_id: light.living_room
+    domain: light
+
+
+```
```

Comment:
Small tweaks to match or generic style guides.

````suggestion
```yaml
# Example automations.yaml entry
alias: Doorbell Pressed
description: "Trigger when Amcrest Button Press Event Fires"
trigger:
  - platform: event
    event_type: amcrest
    event_data:
      event: "CallNoAnswered"
      payload:
        action: "Start"
action:
  - type: flash
    entity_id: light.living_room
    domain: light
```
````